templates/horizon/config/local_settings.py

```diff
@@ -94,12 +94,12 @@
 # memcached set CACHES to something like below.
 # For more information, see
 # https://docs.djangoproject.com/en/1.11/topics/http/sessions/.
-#CACHES = {
-#    'default': {
-#        'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
-#        'LOCATION': '127.0.0.1:11211',
-#    },
-#}
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.memcached.PyMemcacheCache',
+        'LOCATION': '{{.memcachedSvc}}:11211',
```

Comment:
PyMemcacheCache does not require inet6 prefix so this should work for both IPv4 and IPv6.